doc/TSCC.rst

```diff
@@ -1,53 +1,51 @@
 TSCC
 ====
 
-Last update: 2020/08/31
+Last update: 2024/01/25
 
 Official docs
 -------------
 * The `tscc user guide <https://www.sdsc.edu/support/user_guides/tscc.html>`_
 * The `tscc description <https://www.sdsc.edu/services/hpc/hpc_systems.html#tscc>`_
+* THe `tscc 2.0 transitional workshow video <https://youtu.be/U_JGz-sQoV4?si=vFXfDWSIribuTLzd>`_
 
 Logging in
 ----------
-:code:`ssh <user>@tscc.sdsc.edu`
+:code:`ssh <user>@login.tscc.sdsc.edu`
 
-* This will put you on a node such as `tscc-login1.sdsc.edu` or `tscc-login11.sdsc.edu` or `tscc-login2.sdsc.edu`.
-  You can also ssh into  those nodes directly (e.g. if you have :code:`tmux` sessions saved on one of them)
+* This will put you on a node such as `login1.tscc.sdsc.edu` or `login11.tscc.sdsc.edu` or `login2.tscc.sdsc.edu`.
+  You can also ssh into those nodes directly (e.g. if you have :code:`tmux` sessions saved on one of them)
 
 The login nodes are often quite slow because there are too many users on them, and you're not supposed to run code that's
 at all computationally burdensome there. So if you want to use tscc as a workstation, you should immediately try to grab an
-interactive session. :code:`qsub -I -l walltime=<n>:00:00 -l nodes=1:ppn=1 -q home` where :code:`<n>` is the max
-number of hours you plan to work. That will wait till it can find a slot on our home node and then log you into
-that node. (Note: :code:`qsub`'s user interface sucks, if you mistype that command it will hang forever without
-telling you that's what happened. You should see :code:`qsub: waiting for job 22409571.tscc-mgr7.local to start`
-if you typed the command correctly)
+interactive session. :code:`srun --partition=condo --account=ddp268 --pty --nodes=1 --ntasks-per-node=4 -t <n>:00:00 --wait=0 --qos=condo --export=ALL /bin/bash` where :code:`<n>` is the max
```

Comment:
```suggestion
interactive session. :code:`srun --partition=condo --account=ddp268 --pty --nodes=1 --ntasks=1 --cpus-per-task=4 -t <n>:00:00 --wait=0 --qos=condo --export=ALL /bin/bash` where :code:`<n>` is the max
```
gonna update this in accordance with the discussion in https://github.com/gymrek-lab/gymreklab.github.io/pull/59/files#r1546435683